scripts/core/exp-esimd.yml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Unified Runtime Experimental APIs for ESIMD"
+ordinal: "99"
+--- #--------------------------------------------------------------------------
+type: enum
+extend: true
+typed_etors: true
+desc: "Extension enums to $x_device_info_t to support ESIMD."
+name: $x_device_info_t
+etors:
+    - name: ESIMD_SUPPORT_EXP
+      value: "0x2010"
+      desc: "[$x_bool_t] returns true if the device supports ESIMD"
```

Comment:
Do you expect there will be any other additions relating to device ESIMD support in future, such as additional entry-points or extra handle creation data?

Reply:
Thanks for the review! I don't know of any plans or possible features in ESIMD that would require any UR changes. I think it's very unlikely we will add anything else, but if there is something to do now that would prevent some nightmare in the future if we do end up adding something, I'm happy to do it.

Reply:
Okay, since there's no other addidions currently planned I think this device info query could probably be added directly to `scripts/core/device.yml` instead of using the experimental feature mechanism. So far we've only used it for features which are likley to have a lot of API churm before becoming stable.

Reply:
Okay thanks, let me make that change.